### ISSUE TITLE
NoMethodError (undefined method `order')

### ISSUE THREAD

**Issue by YushengLi** [zilkey/active_hash#191]
According to README, I should be able to define class and call `.order` on `ActiveHash::Relation` or the Class itself, but non of the following will work.

```ruby
require 'active_hash'

class Country < ActiveHash::Base
  self.data = [
    {:id => 1, :name => "US"},
    {:id => 2, :name => "Canada"}
  ]
end
```

Calling `.order` on the Class itself will raise `NoMethodError`
```shell
#=> Country.order(id: :desc)

Traceback (most recent call last):
       16: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/exe/bundle:22:in `<top (required)>'
       15: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/friendly_errors.rb:124:in `with_friendly_errors'
       14: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/exe/bundle:30:in `block in <top (required)>'
       13: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/cli.rb:18:in `start'
       12: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
       11: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/cli.rb:27:in `dispatch'
       10: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
        9: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
        8: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
        7: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/cli.rb:465:in `exec'
        6: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/cli/exec.rb:28:in `run'
        5: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/cli/exec.rb:74:in `kernel_load'
        4: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/cli/exec.rb:74:in `load'
        3: from /Users/yusheng/.rbenv/versions/2.5.3/bin/irb:11:in `<top (required)>'
        2: from (irb):8
        1: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/active_hash-3.0.0/lib/active_hash/base.rb:247:in `method_missing'
NoMethodError (undefined method `order' for Country:Class)
```

Calling the method on `ActiveHash::Relation` will also raise `NoMethodError`

```shell
#=> Country.all.order(id: :desc)

Traceback (most recent call last):
       16: from /Users/yusheng/.rbenv/versions/2.5.3/bin/bundle:23:in `load'
       15: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/exe/bundle:22:in `<top (required)>'
       14: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/friendly_errors.rb:124:in `with_friendly_errors'
       13: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/exe/bundle:30:in `block in <top (required)>'
       12: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/cli.rb:18:in `start'
       11: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
       10: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/cli.rb:27:in `dispatch'
        9: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
        8: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
        7: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
        6: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/cli.rb:465:in `exec'
        5: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/cli/exec.rb:28:in `run'
        4: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/cli/exec.rb:74:in `kernel_load'
        3: from /Users/yusheng/.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/bundler-2.0.2/lib/bundler/cli/exec.rb:74:in `load'
        2: from /Users/yusheng/.rbenv/versions/2.5.3/bin/irb:11:in `<top (required)>'
        1: from (irb):9
NoMethodError (undefined method `order' for #<ActiveHash::Relation:0x00007fd15fcf4a08>)
```

### Version Information
Used Version: v3.0.0
Ruby Version: 2.5.3
Rails Version: 5.2.3

I have also tried v2.3.0 with no luck, but if I referenced master branch version directly, the problem gets resolved

**Comment by YushengLi** [comment]
Sorry, I just realized that `AcitveHash::Relation#order` was added in 9ebc83d1a8c65deed9c5ba1a75717ba02d2eec78 on Oct, 2019, which is after version 3.0.0 🙇

**Comment by syguer** [comment]
@YushengLi Yes, yes, you're right. Sorry for late but I've released v3.1.0 just now!
